packages/theming/src/index.ts

```diff
@@ -24,6 +24,7 @@ export { useWindow } from './utils/useWindow';
 export { useText } from './utils/useText';
 export { default as menuStyles } from './utils/menuStyles';
 export { focusStyles, SELECTOR_FOCUS_VISIBLE } from './utils/focusStyles';
+export { StyledBaseIcon } from './styled/StyledBaseIcon';
```

Comment:
Should this just live in `utils`? We have other `...Styles` utilities in there based heavily on `styled-components`. I'm thinking we can avoid this one-off.

Reply:
Yeah that feels better.